app.js

```diff
@@ -243,7 +243,7 @@ function App (el) {
         self.emit('leaveChannel', self.data.activeChannel.name)
         break
       default:
-        console.log('Unrecognized /command: ' + command)
+        console.log('Unrecognized command: ' + commandStr)
```

Comment:
no, exactly. should be `command` to have output
```
Unrecognized command: /hello
```
that's was the idea i mentioned